lib/manageiq/environment.rb

```diff
@@ -19,7 +19,16 @@ def self.manageiq_plugin_setup
         create_database_user
       end
 
-      setup_test_environment
+      setup_test_environment(:task_prefix => 'app:', :root => plugin_root)
+    end
+
+    def self.manageiq_plugin_update
```

Comment:
I was following along  `bin/setup` and `bin/update` in the core, which do `bundle install` and `bundle update`. But 👍 for simplicity. Will drop the update